src/components/mainLayout/youtubeEmbed/youtubeEmbed.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import { IYoutubeEmbed } from "../../../types/components/ComponentsTypes";
+
+import fallback from "../../../assets/components/youtubeEmbed/fallback.png";
+
+import style from "./youtubeEmbed.module.scss";
+
+const YoutubeEmbed = ({ embedId }: IYoutubeEmbed) => {
+	const [hasError, setHasError] = useState(true);
```

Comment:
const [hasError, setHasError] = useState<boolean>(false);

we don't have error in state by default